Sammuel09/Fast-Food-Fast
# UI Review

#### What does this PR do?
- Updates the views of the app

#### Description of Task to be completed?
- create the admin profile page
- create add food items page
- create view food items
- create orders page
- create the user profile page
- create the order history page

#### How should this be manually tested?
After cloning the repo, cd into the folder. Open the HTML pages in your browser to see the changes

#### Any background context you want to provide?
These changes implement feedback received from my LFA

#### What are the relevant pivotal tracker stories?
#160408816, #160408859

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/28958299/45725275-0529d300-bbb2-11e8-8e72-5a477580ac94.png)
